metadata-ingestion/tests/integration/bigquery_v2/test_bigquery.py

```diff
@@ -577,3 +578,146 @@ def test_bigquery_queries_v2_lineage_usage_ingest(
         output_path=mcp_output_path,
         golden_path=mcp_golden_path,
     )
+
+
+@freeze_time(FROZEN_TIME)
+@patch.object(BigQuerySchemaApi, "get_snapshots_for_dataset")
+@patch.object(BigQuerySchemaApi, "get_views_for_dataset")
+@patch.object(BigQuerySchemaApi, "get_tables_for_dataset")
+@patch.object(BigQuerySchemaGenerator, "get_core_table_details")
+@patch.object(BigQuerySchemaApi, "get_datasets_for_project_id")
+@patch.object(BigQuerySchemaApi, "get_columns_for_dataset")
+@patch.object(BigQueryDataReader, "get_sample_data_for_table")
+@patch("google.cloud.bigquery.Client")
+@patch("google.cloud.datacatalog_v1.PolicyTagManagerClient")
+@patch("google.cloud.resourcemanager_v3.ProjectsClient")
+@pytest.mark.parametrize(
+    "use_queries_v2, include_table_lineage, golden_file",
+    [
+        (True, False, "bigquery_mcp_lineage_golden"),
+        (True, True, "bigquery_mcp_full_lineage_golden"),
+        (False, False, "bigquery_mcp_no_queries_v2_golden"),
+        (False, True, "bigquery_mcp_no_queries_v2_full_lineage_golden"),
```

Comment:
While its great to test all combinations here, I believe we can use exactly same golden file for all cases, if `include_usage_statistics` is disabled. Or two golden files even if its not.  I would prefer any of above over multiple golden files that are essentially exactly same. Can you please make the change ?

Reply:
removed